serial_port: "/dev/ttyACM0"
use_network: False
verbose: True
algorithm: 0
dimension: 0
height: 0
frequency: 0
tag_frame_id: "tag_frame"
visualize_anchors: True

